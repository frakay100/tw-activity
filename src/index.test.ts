import * as webhook from "./index";

test("API sheme", () => {
    expect(webhook).toMatchSnapshot();
});
