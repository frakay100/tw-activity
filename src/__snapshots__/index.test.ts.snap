// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`API sheme 1`] = `
Object {
  "api": Object {
    "addSubscriptions": [Function],
    "createWebhook": [Function],
    "deleteSubscriptions": [Function],
    "deleteWebhook": [Function],
    "getSubscriptions": [Function],
    "getWebhooks": [Function],
    "triggerWebhookCRC": [Function],
  },
  "crc": Object {
    "default": [Function],
  },
  "default": Object {
    "addSubscriptions": [Function],
    "createWebhook": [Function],
    "deleteSubscriptions": [Function],
    "deleteWebhook": [Function],
    "getSubscriptions": [Function],
    "getWebhooks": [Function],
    "triggerWebhookCRC": [Function],
  },
  "message": Object {
    "addSubscriptionsMessage": [Function],
    "createWebhookMessage": [Function],
    "deleteSubscriptionsMessage": [Function],
    "deleteWebhookMessage": [Function],
    "getSubscriptionsMessage": [Function],
    "getWebhooksMessage": [Function],
    "triggerWebhookCRCMessage": [Function],
  },
}
`;
